Tasks/KubernetesV1/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 1,
```

Comment:
@bansalaseem should it be a minor version upgrade instead? Cause this contains a new feature